README.md

```diff
@@ -69,6 +69,9 @@ Options include:
 {
   sparse: true, // only download data on content feed when it is specifically requested
   sparseMetadata: true // only download data on metadata feed when requested
+  metaStorageCacheSize: 65536 // how many entries to use in the metadata hypercore's LRU cache
```

Comment:
can you call this metadataStorageCacheSize?

Reply:
Dine